src/Pages/Detail/Components/HeadModal.js

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useEffect } from "react";
+import { AiOutlineLeft, AiOutlineRight } from "react-icons/ai";
+import { useSelector } from "react-redux";
+import Styled from "styled-components";
+const HeadModal = ({ handleModal, HandleSlick, slickWidth, currentPage }) => {
```

Comment:
```suggestion
import Styled from "styled-components";

const HeadModal = ({ handleModal, handleSlick, slickWidth, currentPage }) => {
```
- 구분 되는 내용끼리는 반드시 한줄 띄워주세요
- 함수 이름 대문자로 시작하고 있습니다. 소문자로 바꿔주세요